doc/getting_started.rst

```diff
@@ -235,7 +235,41 @@ Now you can use **Lua Fun**:
     2 c
     one
 
-Please note that functions support multireturn.
+Please note that functions support multireturn. It means that some of them may
```

Comment:
Just thought: is not it too hard to be in the 'Getting started' section?

Reply:
I think yes. There is another appropriate section - 'Under the hood', but it looks as additional information for advanced users. Note about multireturn is for everyone who use `span` or `partititon`.